corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -153,42 +176,69 @@ hqDefine("geospatial/js/geospatial_map", [
                 return map;
             };
 
-            self.removeMarkers = function () {
-                // Remove markers
-                _.each(caseModels(), function(currCase) {
+            self.removeCaseMarkers = function () {
+                _.each(caseModels(), function (currCase) {
                     if (currCase.marker) {
                         currCase.marker.remove();
                     }
                 });
                 caseModels([]);
             };
 
+            self.removeUserMarkers = function () {
+                _.each(userModels(), function (currUser) {
+                    if (currUser.marker) {
+                        currUser.marker.remove();
+                    }
+                });
+                userModels([]);
+            };
```

Comment:
we could make a generic function for `removeCaseMarkers` and `removeUserMarkers`?

Reply:
[ef300ef](https://github.com/dimagi/commcare-hq/pull/33488/commits/ef300ef11e125ef447fd01d1762338d3ec6cbf52)